src/v3/src/util/htmlContentParserUtils.tsx

```diff
@@ -10,7 +10,7 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { Link, Typography } from '@okta/odyssey-react-mui-legacy';
+import { Link, Typography } from '@mui/material';
```

Comment:
`Link` and `Typography` components get arbitrary props spread into them in these functions so it's safer to use MUI version of the components